internal/bminventory/inventory.go

```diff
@@ -5866,3 +5878,55 @@ func isBaremetalBinaryFromAnotherReleaseImageRequired(cpuArchitecture, version s
 		featuresupport.IsFeatureSupported(version,
 			models.FeatureSupportLevelFeaturesItems0FeatureIDARM64ARCHITECTUREWITHCLUSTERMANAGEDNETWORKING)
 }
+
+// updateMonitoredOperators checks the content of the installer configuration and updates the list
+// of monitored operators accordingly. For example, if the installer configuration uses the
+// capabilities mechanism to disable the console then the console operator is removed from the list
+// of monitored operators.
+func (b *bareMetalInventory) updateMonitoredOperators(tx *gorm.DB, cluster *common.Cluster) error {
+	// Get the complete installer configuration, including the overrides:
+	installConfigData, err := b.installConfigBuilder.GetInstallConfig(cluster, false, "")
+	if err != nil {
+		return err
+	}
+	var installConfig installcfgdata.InstallerConfigBaremetal
```

Comment:
Is it always InstallerConfigBaremetal even if the platform is not baremetal?

Reply:
Yes @avishayt, that is the name that we decided to use for our replica of the `InstallerConfig` type of the installer: https://github.com/openshift/assisted-service/blob/1add0c2637bca84010269c12ba71ab3f74316b57/internal/installcfg/installcfg.go#L120-L151

Reply:
OK got it, thanks.